src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -24,6 +25,9 @@ const AnonymousChannelListScreen = ({route}) => {
     goToPostDetailScreen,
     goToContactScreen
   } = useAnonymousChannelListScreenHook();
+  const ref = React.useRef(null);
+
+  useScrollToTop(ref);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `useScrollToTop(ref)` is correct and it's a good practice to use React's useRef hook to create a ref for the FlatList. This allows the list to be scrolled back to the top when needed, improving user experience.

However, I would suggest adding a null check before using `useScrollToTop(ref)`. This will ensure that the ref is defined before it's used, preventing potential errors.
```diff
-  useScrollToTop(ref);
+  if (ref.current) {
+    useScrollToTop(ref);
+  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->